src/py/flwr/simulation/ray_transport/ray_actor.py

```diff
@@ -58,27 +52,25 @@ def terminate(self) -> None:
 
     def run(
         self,
-        client_fn: ClientFn,
-        job_fn: JobFn,
+        app_fn: FlowerFn,
```

Comment:
Should we call this `client_app_fn` to distinguish it from `server_app_fn`?
```suggestion
        client_app_fn: FlowerFn,
```